### PR TITLE
Remove redundant What's New for Auto Play

### PR DESCRIPTION
**Pull request by ashiagr**
## Description
This removes redundant code for `What's New` for `Auto Play`

## Testing Instructions
CI should be 🟢

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/Automattic/pocket-casts-android/milestone/45">7.57 ❄️</a>. The due date for this milestone has already passed.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: modules/services/localization/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -1409,10 +1409,6 @@
 
     <string name="settings_advanced_sync_on_metered">Sync on Metered Networks</string>
     <string name="settings_advanced_sync_on_metered_summary">Permits automatic background sync on metered networks. This is enabled by default and has no effect if background refresh is turned off.</string>
-    <string name="whats_new_autoplay_title">Autoplay is here!</string>
-    <string name="whats_new_autoplay_body">If you start an episode while Up Next is empty, Autoplay will continue playing episodes from that podcast or list.</string>
-    <string name="whats_new_autoplay_enable_button">Enable it now</string>
-    <string name="whats_new_autoplay_maybe_later_button">Maybe later</string>
```
Comment (MiSikora) [inline modules/services/localization/src/main/res/values/strings.xml:1415]: Just to clarify - other languages are delete automatically with translations sync?

**Review by MiSikora — APPROVED** [review]

File: modules/services/localization/src/main/res/values/strings.xml (hunk 1 above)
Comment (ashiagr) [reply modules/services/localization/src/main/res/values/strings.xml:1415]: That's correct. Thanks for double-checking. 👍

**Review by mebarbosa — APPROVED** [review]
LGTM

**Pull request merged**